### PR TITLE
Removes redundant and misleading alt-titles, adds title_blurbs to a lot that remain, adds one or two more alt-titles

### PR DESCRIPTION
**Pull request by SingingSpock**
## About The Pull Request

Trims out a handful of alt-titles that were misleading and/or redundant, and adds title_blurb text where missing to most of the ones that remain in files I touched, to help establish what each alt-title is for. (I then realized that nothing shows the title_blurb in-game, but that's a problem for another PR). The departmental senior titles were also given a desc that indicates their actual role in our current SOP (including Senior Researcher, whose desc was previously literally "Lorem Ipsum." There's also a couple alt-titles, most notably Trainee EMT, since there wasn't a good role for that previously. Exact details of the changes are too long to put in here, but it's pretty easily readable. The most significant alteration was changing almost all the titles for the senior medical position, removing "Head Nurse" (because that makes no sense whatsoever as a role of authority that doctors would look towards for advice) and making "Senior Physician" the primary title.

## Why It's Good For The Game

Alt-titles were often just a dozen ways to say kinda the same thing. In the worst cases, they were actively confusing (e.g. Counselor as an alt-title for both Chaplain and Psychiatrist) or not indicative at all of the original job (e.g. Director of Operations for Captain). This leaves each alt-title in non-service departments having a distinct role that they fill, indicated by title_blurb text (which I will work on fixing next since I noticed it isn't actually shown in-game). Additionally, a couple of niches weren't previously covered by alt-titles (such as Trainee EMT), so those were added.

## Changelog

:cl: SingingSpock
tweak: Alt-titles were trimmed down and more description given to many remaining ones. A couple more were added.
/:cl:

### CODE REVIEW

**Comment by SingingSpock** [comment]
I don't know why the linter is still complaining, I re-added the trailing newlines to those files

File: code/modules/jobs/job_types/station/security/detective.dm [hunk 1]
```diff
@@ -76,9 +67,4 @@
 	name = OUTFIT_JOB_NAME("Forensic technician")
 	head = null
 	suit = /obj/item/clothing/suit/storage/forensics/blue
-
-/datum/outfit/job/station/detective/vice
-	name = OUTFIT_JOB_NAME("Vice Investigator")
-	head = null
-	uniform = /obj/item/clothing/under/hawaiian
-	suit = null
+
```
Comment (timothyteakettle) [inline code/modules/jobs/job_types/station/security/detective.dm:70]: this is not an empty newline

File: code/modules/jobs/job_types/station/science/senior_researcher.dm [hunk 2]
```diff
@@ -45,7 +55,4 @@
 	satchel_one = /obj/item/storage/backpack/satchel/tox
 	messenger_bag = /obj/item/storage/backpack/messenger/tox
 	dufflebag = /obj/item/storage/backpack/dufflebag/sci
-
-/datum/outfit/job/station/scientist/xenobiologist
-	name = OUTFIT_JOB_NAME("Xenobiologist")
-	id_type = /obj/item/card/id/science/xenobiologist
+
```
Comment (timothyteakettle) [inline code/modules/jobs/job_types/station/science/senior_researcher.dm:58]: this is also not an empty newline

**Review by timothyteakettle — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/station/command/head_of_personnel.dm [hunk 3]
```diff
@@ -32,7 +32,7 @@
 						manage the Supply department, through the Quartermaster. In addition, the Head of Personnel oversees the personal accounts \
 						of the crew, including their money and access. If necessary, the Head of Personnel is first in line to assume Acting Command."
 	alt_titles = list(
-		"Crew Resources Officer" = /datum/prototype/struct/alt_title/cro,
+		"First Officer" = /datum/prototype/struct/alt_title/fo,
```
Comment (silicons) [inline code/modules/jobs/job_types/station/command/head_of_personnel.dm:35]: don't remove CRO

File: code/modules/jobs/job_types/station/engineering/senior_engineer.dm [hunk 4]
```diff
@@ -30,18 +30,15 @@
...
-		"Artificer-Adept" = /datum/prototype/struct/alt_title/artificer_adept
+		"Engineering Training Specialist" = /datum/prototype/struct/alt_title/engi_trainer
 		)
 
-/datum/prototype/struct/alt_title/artificer_adept
-	title = "Artificer-Adept"
-	background_allow = list(
-		/datum/lore/character_background/faction/naramadiguilds
-	)
-	background_enforce = TRUE
```
Comment (silicons) [inline code/modules/jobs/job_types/station/engineering/senior_engineer.dm:44]: don't remove this without checking with lore people

File: code/modules/jobs/job_types/station/engineering/station_engineer.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /datum/role/job/station/engineer
 	id = JOB_ID_STATION_ENGINEER
-	title = "Station Engineer"
+	title = "Engineer"
```
Comment (silicons) [inline code/modules/jobs/job_types/station/engineering/station_engineer.dm:3]: i don't really see the need to shorten this to engineer

File: code/modules/jobs/job_types/station/engineering/station_engineer.dm [hunk 6]
```diff
@@ -30,51 +30,42 @@
...
+/datum/prototype/struct/alt_title/damage_tech
+	title = "Damage Control Technician"
+	title_blurb = "A Damage Control Technician specializes in repairing a damaged facility as quickly as possible, usually possessing good skills with both atmospherics and quick repair work."
 
-/datum/prototype/struct/alt_title/artificer
-	title = "Artificer"
-	background_allow = list(
-		/datum/lore/character_background/faction/naramadiguilds
-	)
-	background_enforce = TRUE
```
Comment (silicons) [inline code/modules/jobs/job_types/station/engineering/station_engineer.dm:77]: don't remove these without asking lore people

File: code/modules/jobs/job_types/station/medical/chemist.dm [hunk 7]
```diff
@@ -28,17 +28,9 @@
...
+		"Pharmacologist" = /datum/prototype/struct/alt_title/pharmacologist
 		)
 
-/datum/prototype/struct/alt_title/biochemist
-	title = "Biochemist"
-	background_allow = list(
-		/datum/lore/character_background/faction/onkhera_necropolis
-	)
-	background_enforce = TRUE
-
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/chemist.dm:41]: don't remove this without checking with lore

File: code/modules/jobs/job_types/station/medical/head_nurse.dm [hunk 8]
```diff
@@ -23,44 +23,28 @@
...
+		"Medical Training Specialist" = /datum/prototype/struct/alt_title/medical_specialist
 	)
 
-/datum/prototype/struct/alt_title/biotechnical_advisor
-	title = "Biotechnical Advisor"
-	title_outfit = /datum/outfit/job/station/medical_doctor // todo: add OSSNECRO outfits.
-	background_allow = list(
-		/datum/lore/character_background/faction/onkhera_necropolis
-	)
-	background_enforce = TRUE
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/head_nurse.dm:47]: dont remove this without checking with lore

File: code/modules/jobs/job_types/station/medical/medical_doctor.dm [hunk 9]
```diff
@@ -32,33 +32,17 @@
 		"Nurse" = /datum/prototype/struct/alt_title/nurse,
 		"Virologist" = /datum/prototype/struct/alt_title/virologist,
 		"Medical Resident" = /datum/prototype/struct/alt_title/doctor/resident,
-		"Medical Intern" = /datum/prototype/struct/alt_title/doctor/intern,
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/medical_doctor.dm:35]: don't remove this; much like we have cadet vs jr officer

File: code/modules/jobs/job_types/station/medical/medical_doctor.dm [hunk 10]
```diff
@@ -32,33 +32,17 @@
...
 
 // Medical Doctor Alt Titles
 
-/datum/prototype/struct/alt_title/biotechnician
-	title = "Biotechnician"
-	title_outfit = /datum/outfit/job/station/medical_doctor/surgeon //todo: add OSSNECRO outfits
-	background_allow = list(
-		/datum/lore/character_background/faction/onkhera_necropolis
-	)
-	background_enforce = TRUE
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/medical_doctor.dm:39]: dont remove this without checking with lore

File: code/modules/jobs/job_types/station/medical/paramedic.dm [hunk 11]
```diff
@@ -25,32 +25,23 @@
...
+		"Trainee Emergency Medical Technician" = /datum/prototype/struct/alt_title/emt_trainee
 		)
 
-/datum/prototype/struct/alt_title/biotech_responder
-	title = "Biotechnical Responder"
-	title_outfit = /datum/outfit/job/station/paramedic/emt  //TODO: add OSSNECRO outfits.
-	background_allow = list(
-		/datum/lore/character_background/faction/onkhera_necropolis
-	)
-	background_enforce = TRUE
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/paramedic.dm:42]: dont remove this without checking with lore

File: code/modules/jobs/job_types/station/medical/paramedic.dm [hunk 12]
```diff
@@ -25,32 +25,23 @@
...
 	)
 	outfit_type = /datum/outfit/job/station/paramedic
-	desc = "A Paramedic is primarily concerned with the recovery of patients who are unable to make it to the Medical Department on their own. \
+	desc = "A Paramedic is primarily concerned with the stabilization and recovery of patients who are unable to make it to the Medical Department on their own. \
 						They may also be called upon to keep patients stable when Medical is busy or understaffed."
 	alt_titles = list(
 		"Emergency Medical Technician" = /datum/prototype/struct/alt_title/emt,
-		"Medical Responder" = /datum/prototype/struct/alt_title/medical_responder,
-		"Biotechnical Responder" = /datum/prototype/struct/alt_title/biotech_responder
+		"Trainee Emergency Medical Technician" = /datum/prototype/struct/alt_title/emt_trainee
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/paramedic.dm:32]: way too wordy; a trainee should just be intern

File: code/modules/jobs/job_types/station/medical/psychiatrist.dm [hunk 13]
```diff
@@ -24,9 +24,7 @@
 					ails the mentally unwell, frequently under Security supervision. They understand the effects of various psychoactive drugs."
 	alt_titles = list(
 		"Psychologist" = /datum/prototype/struct/alt_title/psychologist,
-		"Psychoanalyst" = /datum/prototype/struct/alt_title/psychologist/psychoanalyst,
-		"Counselor" = /datum/prototype/struct/alt_title/counselor,
-		"Therapist" = /datum/prototype/struct/alt_title/therapist
```
Comment (silicons) [inline code/modules/jobs/job_types/station/medical/psychiatrist.dm:29]: curious
why are we removing therapist but adding counselor

latter sounds even less like a proper medical role

File: code/modules/jobs/job_types/station/science/roboticist.dm [hunk 14]
```diff
@@ -25,19 +25,10 @@
...
+		"Mechatronic Technician" = /datum/prototype/struct/alt_title/mech_tech
 		)
 
-/datum/prototype/struct/alt_title/artificer_specialist
-	title = "Artificer-Specialist"
-	background_allow = list(
-		/datum/lore/character_background/faction/naramadiguilds
-	)
-	background_enforce = TRUE
-
```
Comment (silicons) [inline code/modules/jobs/job_types/station/science/roboticist.dm:40]: blah blah lore

File: code/modules/jobs/job_types/station/science/roboticist.dm [hunk 15]
```diff
@@ -25,19 +25,10 @@
 	desc = "A Roboticist maintains and repairs the station's synthetics, including crew with prosthetic limbs. \
 						They can also assist the station by producing simple robots and even pilotable exosuits."
 	alt_titles = list(
-		"Biomechanical Engineer" = /datum/prototype/struct/alt_title/biomech,
-		"Mechatronic Engineer" = /datum/prototype/struct/alt_title/mech_tech,
-		"Prosthetists" = /datum/prototype/struct/alt_title/prosthetists,
-		"Artificer-Specialist" = /datum/prototype/struct/alt_title/artificer_specialist
+		"Biomechanical Technician" = /datum/prototype/struct/alt_title/biomech,
+		"Mechatronic Technician" = /datum/prototype/struct/alt_title/mech_tech
```
Comment (silicons) [inline code/modules/jobs/job_types/station/science/roboticist.dm:29]: engineer sounds cooler for both tbh; why are we changing it?

File: code/modules/jobs/job_types/station/security/warden.dm [hunk 16]
```diff
@@ -34,15 +34,16 @@
 						Armoury gear in a crisis, and retrieving it when the crisis has passed. In an emergency, the Warden may be called upon to direct the \
 						Security Department as a whole."
 	alt_titles = list(
-		"Jailor" = /datum/prototype/struct/alt_title/warden/jailor,
+		"Brig Overseer" = /datum/prototype/struct/alt_title/warden/overseer,
```
Comment (silicons) [inline code/modules/jobs/job_types/station/security/warden.dm:37]: ngl this is such a silly title but whatever idc

File: code/modules/jobs/job_types/station/supply/cargo_technician.dm [hunk 17]
```diff
@@ -24,25 +24,9 @@
...
-	)
-	background_enforce = TRUE
-
-/datum/prototype/struct/alt_title/logisticai
-	title = "Logisticai"
-	background_allow = list(
-		/datum/lore/character_background/faction/naramadiguilds
-	)
-	background_enforce = TRUE
-
```
Comment (silicons) [inline code/modules/jobs/job_types/station/supply/cargo_technician.dm:45]: blah blah lore

File: code/modules/jobs/job_types/station/supply/quartermaster.dm [hunk 18]
```diff
@@ -31,18 +31,14 @@
 	ideal_character_age = 40
 
 	outfit_type = /datum/outfit/job/station/quartermaster
-	desc = "The Quartermaster manages the Supply department, checking cargo orders and ensuring supplies get to where they are needed."
+	desc = "The Quartermaster manages the Supply department under the Head of Personnel, checking cargo orders and ensuring supplies get to where they are needed."
 	alt_titles = list(
 		"Supply Chief" = /datum/prototype/struct/alt_title/supply_chief,
-		"Logisticai-Adept" = /datum/prototype/struct/alt_title/logisticai_adept
```
Comment (silicons) [inline code/modules/jobs/job_types/station/supply/quartermaster.dm:37]: blah blah lore

**Review by silicons — CHANGES_REQUESTED** [review]
blah blah subjective nitpicks

**Review by BlueWildrose — COMMENTED** [review]
Also, it'd be neat if the shaft miner had some alt titles to do with salvaging. Chances are that might become their default titles eventually anyways.
Salvage Technician, Salvage Specialist, or something. Maybe both.

**Review by silicons — APPROVED** [review]

**Comment by silicons** [comment]
@timothyteakettle
